url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/inport/ScrollProperty.java

```diff
@@ -0,0 +1,75 @@
+package net.nexon.vindictus.itemcomparer.inport;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+public class ScrollProperty {
+
+	private double enthu = 12.0;
+	private double force = 3.0;
+	private double armadi = 1.0;
+	private double senti = 3.0;
+	private double tut = 3.0;
+	private double res = 0.4;
+	private double master = 0.5;
+	private double wb = 0.0;
+
+	public ScrollProperty() {
+		Properties props = new Properties();
+		try {
+			BufferedInputStream stream = new BufferedInputStream(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *stream*. Execution paths that do not contain closure statements were detected. Ensure *stream* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).